org.civicrm.reqangular/test/services/format-currency.spec.js

```diff
@@ -0,0 +1,116 @@
+/* eslint-env amd, jasmine */
```

Comment:
Should we add more edge cases like 1.01, "1.101", 0, -1, -0, "xxx", "100.000.00" etc.

Reply:
Re-written test cases.